comptests/motr/test_motr_di.py

```diff
@@ -266,3 +273,59 @@ def test_data_block_corruption_one_by_one(self):
             # Read the data using m0cp utility
             self.m0cat_md5sum_m0unlink(bsize_list, count_list, layout_ids, object_list,
                                        client_num=client_num)
+
+    @pytest.mark.skip(reason="Test incomplete without teardown")
+    @pytest.mark.tags("TEST-42910")
+    @pytest.mark.motr_di
+    def test_m0cp_block_corruption_m0cat_degraded_mode(self):
+        """
+        Corrupt data block using m0cp and reading from object with m0cat should error.
+        -s 4096 -c 10 -o 1048583 /root/infile -L 1
+        -s 4096 -c 1 -o 1048583 /root/myfile -L 1 -u -O 0
+        -o 1048583 -s 4096 -c 10 -L 1 /root/dest_myfile
+        """
+        count_list = ['8']
+        bsize_list = ['4K']
+        layout_ids = ['1']
+        proc_restart_delay = di_cfg['wait_time_m0d_restart']
+        process = PID_WATCH_LIST[0]
+        logger.info("STARTED: m0cp, corrupt workflow in healthy state")
+        infile = TEMP_PATH + 'input'
+        outfile = TEMP_PATH + 'output'
+        node_pod_dict = self.motr_obj.get_node_pod_dict()
+        motr_client_num = self.motr_obj.get_number_of_motr_clients()
+        object_id_list = []
+        for client_num in range(motr_client_num):
+            for node in node_pod_dict:
+                object_id = str(self.system_random.randint(1, 1024 * 1024)) + ":" + \
+                            str(self.system_random.randint(1, 1024 * 1024))
+                for b_size, cnt_c, layout in zip(bsize_list, count_list, layout_ids):
+                    self.motr_obj.dd_cmd(
+                        b_size, cnt_c, infile, node)
+                    object_id_list.append(object_id)
+                    self.motr_obj.cp_cmd(
+                        b_size, cnt_c, object_id,
+                        layout, infile, node, client_num)
+        # Degrade the setup by killing the m0d process
+        data_pod_list = self.motr_obj.master_node_list[0].select_random_pod_container(
+            POD_NAME_PREFIX, f"{MOTR_CONTAINER_PREFIX}-001")
+        pod_name, container = secrets.choice(data_pod_list)
```

Comment:
We are already selecting random pod and container, no need for secrets.choice

Reply:
done